ui/app/governance/views/virtual-key-dialog.tsx

```
@@ -309,6 +343,36 @@ export default function VirtualKeyDialog({ virtualKey, teams, customers, onSave,
 
             <DottedSeparator className="mb-5 mt-6" />
 
+            {/* DBKey Selection */}
+            <div className="space-y-2">
+              <div className="flex items-center gap-2">
+                <label className="text-sm font-medium">Allowed Keys</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)